static/scripts/upload_files.js

```diff
@@ -0,0 +1,29 @@
+﻿async function uploadTestFile() {
+    try {
+        // Step 1: Fetch the signed URL from the Go API
+        const response = await fetch("https://your-api.com/get-signed-url");
```

Comment:
should be a relative path `fetch("/get-signed-url")` (though I admit I have to fix this in other places in code as well)